ctypesgen/printer_python/printer.py

```diff
@@ -144,47 +148,20 @@ def print_preamble(self):
 
     def _copy_preamble_loader_files(self, path):
         if os.path.isfile(path):
-            abspath = os.path.abspath(path)
-            dst = os.path.dirname(abspath)
+            dst = os.path.dirname(os.path.abspath(path))
         else:
             error_message(
                 "Cannot copy preamble and loader files",
                 cls="missing-file",
             )
             return
 
-        c_preamblefile = f"{dst}/ctypes_preamble.py"
+        c_preamblefile = join(dst, "ctypes_preamble.py")
         if os.path.isfile(c_preamblefile):
             return
 
-        pointer = """def POINTER(obj):
-    p = ctypes.POINTER(obj)
-
-    # Convert None to a real NULL pointer to work around bugs
-    # in how ctypes handles None on 64-bit platforms
-    if not isinstance(p.from_param, classmethod):
-
-        def from_param(cls, x):
-            if x is None:
-                return cls()
-            else:
-                return x
-
-        p.from_param = classmethod(from_param)
-
-    return p
-
-"""
```

Comment:
I'll revert the changes to `--no-embed-premable` given that it's not related to strings and the way it was before is intentional, though I admittedly don't understand yet how this monkeypatch happens to fix #86.